docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,319 @@ but the format and content is still evolving.
+  Each workload deploys a separate microservice,
+  which can be deployed at different times using different tools.
+
+* KLT integrates with continuous delivery tools
+  to insures that a `KeptnApp` is not deployed
+  if it does not meet the user-defined requirements
+  for all the associated workloads.
+  It also exposes metrics to evaluate the success/status of a deployment.
+
+* KLT provides an operator
+  that can observe and orchestrate application-aware workload life cycles.
+  This operator leverages Kubernetes webhooks
+  and extends the Kubernetes scheduler
+  to support pre- and post-deployment hooks.
+  When the operator detects a new version of a service
+  (implemented as a Kubernetes
+  [Workload](https://kubernetes.io/docs/concepts/workloads/)),
+  it can execute pre- and post-deployment evaluations and tasks
+  using Kubernetes capabilities..
+
+* KLT provides extensive observability data
+  using OpenTelementry and Prometheus
+  rather than storing the data in a special Keptn database.
+  This data can be displayed using Grafana and Jaeger
+  or the dashboard of your choice.
+
+For in-depth information about KLT components
+and how they work, see the
+[Architecture](../../concepts/architecture)
+section.
+
+## Disposition of Keptn v1 components in KLT
+
+To help you wrap your mind around the migration process,
+this lists Keptn v1 components
+and identifies their possible disposition when you migrate to KLT.
+
+### project
+
+A Keptn v1 project represents an arbitrary, higher-level grouping of services.
+A project is defined by a
+[shipyard.yaml](https://keptn.sh/docs/1.0.x/reference/files/shipyard/)
+file.
+
+KLT does not recognize projects.
+Some Keptn v1 projects may translate into `KeptnApp` resources
+but many will not.
+A project has a 1:1 mapping to a Git repo,
+but, since few applications are stored in a mono-repo,
+it is difficult to implement a project-as-application paradigm
+
+In general, the Keptn v1 project is a useful as a reference list
+when migrating to KLT
+but does not directly translate to a KLT resource.
+
+### service
+
+A Keptn v1 service models a smaller chunk of a project.
+Most projects include many services.
+In a micro-services world,
+a service may represent a micro-service
+but it could instead be a wrapper for something else,
+like "the entire public website"
+
+KLT does not have the concept of a service.
+When migrating to KLT,
+you need to analyze what each service is doing
+and translate that into an appropriate resource.
+The closest analogy is a Kubernetes
+[workload](https://kubernetes.io/docs/concepts/workloads/)
+but some services may be translated into
+[KeptnTaskDefinition](../../yaml-crd-ref/app.md)
+or other resources.  See
+[Working with Keptn tasks](../../implementing/tasks/)
+for more information.
+
+For example:
+
+* A Keptnv1 service that runs chaos or load tests
+  can probably be translated into
+  a `KeptnTask` using the `container-runner`.
```

Comment:
maybe here we can link the container-runner to its documentation page.

Reply:
It is already a link, actually ;)